### ISSUE TITLE
AMP covering up all of my content by moving an element and screwing up it's z index

### ISSUE THREAD

**Issue by nikse** [ampproject/amphtml#17995]
**Please only file bugs/feature requests for AMP here.**

- If you have questions about how to use AMP or other general questions about AMP please ask them on Stack Overflow under the AMP HTML tag instead of filing an issue here: http://stackoverflow.com/questions/tagged/amp-html
- If you have questions/issues related to Google Search please ask them in Google's AMP forum instead of filing an issue here: https://goo.gl/utQ1KZ

If you have a bug or feature request for AMP please fill in the following template.  Delete everything except the headers (including this text).

## What's the issue?
You started moving my fixed position background into it's own element, removing the cascading class structure and css and putting your own (very high) zindex on the element thus COVERING ALL OF MY CONTENT

Briefly describe the bug/feature request.
No one can read any content on amp links because you are moving a freaking element in my webpage and covering all of my content

## How do we reproduce the issue?
You can't because I am fixing it because we are a media site and we run on content and brand.

If this is a bug please provide a public URL and ideally a reduced test case (e.g. on jsbin.com) that exhibits only your issue and nothing else.  Then provide step-by-step instructions for reproducing the issue:

1) Again, you can't reproduce on my site because I have to stumble upon my site being broken at midnight and fix it
2) have a fixed element with a low z index as a background of a webpage and load it in amp and watch as amp ruins the webpage

If this is a feature request you can use this section to point to a prototype/mockup that will help us understand the request.

## What browsers are affected?
At least safari. "In order to improve scrolling performance in Safari, we now move the element to a fixed positioning layer: "

All browsers? Some specific browser? What device type?

## Which AMP version is affected?

Is this a new issue? Or was it always broken? Paste your AMP version. You can find it in the browser dev tools.
New version

**Comment by nikse** [comment]
<img width="1245" alt="screen shot 2018-09-11 at 11 36 22 pm" src="https://user-images.githubusercontent.com/1050982/45401409-7996cc00-b61e-11e8-8aef-1fe858870090.png">

**Comment by cathyxz** [comment]
Possibly related to https://github.com/ampproject/amphtml/pull/16049, which went into release [1536699515199](https://github.com/ampproject/amphtml/releases/tag/1536699515199)? 

/cc @jridgewell

**Comment by nikse** [comment]
please let me know if you need any other information or any html snippets. happy to pass along and/or get a private staging env set up for you

**Comment by jridgewell** [comment]
Yup, this is because of #16049.

This will be fixed by the [Shadow DOM transfer](https://github.com/ampproject/amphtml/pull/16642) technique, which is in QA testing now. In the meantime, the easiest fix would be to increment your zIndex by 10000 (hacky).

I'm curious why you're not just using a background image, though?

**Comment by nikse** [comment]
It was actually to support different skinning techniques for advertisers, had to ask around to see if we still needed to support it and we do not so i moved the bg to html and removed fixed element.

Still pretty worrying to have a fixed element moved up on top of all other content

**Comment by jridgewell** [comment]
> Still pretty worrying to have a fixed element moved up on top of all other content

Agreed. The transfer is a technical limitation of scrolling in iOS, and we felt it better than fixed-positions jittering during scrolling. But we're extremely excited for the Shadow DOM technique, which'll fix a few of these issues.

**Comment by brianjihoonlee** [comment]
+1 https://github.com/ampproject/amphtml/pull/16049 
Also causing AMP Consent components impossible to interact with and readers are unable to read content

**Comment by jridgewell** [comment]
@brianjihoonlee: Are you also using a large fixed-position background element to display an image?

**Comment by cipix2000** [comment]
👍 I also hit this bug with a div, positioned fixed, fully transparent and with pointer-events: none. When   moving the element the "pointer-events: none;" was not preserved thus making the AMP page unusable on iOS devices.

**Comment by erwinmombay** [comment]
@jridgewell can you set priority level, just so we know urgency. (i dont know how to assess but im guessing p0)
